src/content/en/updates/2015/03/push-notifications-on-the-open-web.md

```diff
@@ -837,9 +802,9 @@ each approach has pro's and con's.
 
 ### Why do I need a gcm\_sender\_id?
 
-This is required so that Chrome can make use of the Google Cloud Messaging (GCM)
-API. The goal is to use the Web Push Protocol when the standard is finalised and
-GCM can support it.
+This is required so that Chrome, Opera for Android and the Samsung Browser can
+make use of the Firebase Cloud Messaging (FCM) API. The goal is to use the
+Web Push Protocol when the standard is finalised and FCM can support it.
```

Comment:
What is HTTP Push? This is an old article, I just did a sweep to change GCM to FCM.